src/constants.ts

```diff
@@ -192,7 +192,7 @@ export enum ComponentsImportance {
   Grid = 0,
   Highlight = 5,
   Figure = 10,
-  Dropdown = 12,
+  Dropdown = 16,
   ScrollBar = 15,
```

Comment:
Change the order of the elements in the enum to keep it sorted + we should probably add dropdowns in the test `Z-indexes of the various spreadsheet components`

Reply:
Updated changes please check.